### PR TITLE
Added quick start pages and Open CAS Linux Admin Guide

### PR DESCRIPTION
**Pull request by felipe-barajas**
Squashed previous commits into this single commit. Also closed previous pull request as it was getting too difficult to review changes.

### CODE REVIEW

File: pages/open_cas_linux/guide_product_specs_requirements.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+| Software         | The following prerequisite software must be installed prior to Open CAS Linux installation:  {::nomarkdown}<ul><li>bzip2</li><li>tar</li><li>sed</li><li>make</li><li>gcc</li><li>kernel-devel</li><li>kernel-headers</li><li>python2</li><li>lsblk</li><li>lsof</li><li>argparse (python module)</li></ul>{:/}                         |
+| Power Management | Power Management Suspend (S3) and Hibernate (S4) states must be disabled.                                                                                                                                                                                                                                                               |
+
+
+**NOTE:** The cache device logical block size must be smaller than or equal to the logical
+block size of the core storage device. 60GB or larger capacity is recommended.
```
Comment (robertbaldyga) [inline pages/open_cas_linux/guide_product_specs_requirements.md]: I'd remove minimum capacity recommendation. It depends on many factors and it's hard to tell what is the reasonable minimum without knowing particular configuration and workload. The 60GB minimum seems very arbitrary.

File: pages/open_cas_linux/guide_running.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+running, the user can easily configure and start cache instances. Refer to the
+guidance in [**Configuring CAS**](/guide_configuring.html) for configuring the file *utils/opencas.conf*, then
+follow these steps:
+
+1. Edit and configure caches and cores as desired via the
+*opencas.conf* file.
+
+2. Execute the following:
+
+>   \# utils/casctl init
```
Comment (robertbaldyga) [inline pages/open_cas_linux/guide_running.md]: The `casctl` is now installed by `make install` so there is no need to point to `utils/` directory.

File: pages/open_cas_linux/guide_running.md [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+restarted.
+
+Disabling Open CAS Linux
+--------------------
+
+In order to disable Open CAS Linux on one or more devices, the user should perform
+either of the following operations:
+
+1.  Stop caching via casctl stop (--flush as appropriate). Remove or comment
+    devices from the *opencas.conf* file. Perform system restart.
```
Comment (robertbaldyga) [inline pages/open_cas_linux/guide_running.md]: The system restart is not needed. In both cases.

File: pages/open_cas_linux/guide_terminology.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+| dirty data         | This refers to data that is modified within the cache but not modified in main memory.                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| guest              | An operating system running on a Virtual Machine (VM) environment.                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (robertbaldyga) [inline pages/open_cas_linux/guide_terminology.md]: Words "guest", "host" and "hypervisior" do not occur on any page of this documentation. (At least not in meaning described here).

File: pages/open_cas_linux/guide_terminology.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+| lazy write         | The process of mirroring to primary storage. See also write-back.                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
+| NAS                | Network-attached storage. File-level data storage (such as fixed disk and magnetic tape drives) that are directly linked to a storage area network or other network.                                                                                                                                                                                                                                                                                                                                                                         |
```
Comment (robertbaldyga) [inline pages/open_cas_linux/guide_terminology.md]: Acronym NAS is also never used in this documentation.

File: pages/open_cas_linux/guide_terminology.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+| IO                 | Abbreviation for input/output as it relates to the flow of data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| lazy write         | The process of mirroring to primary storage. See also write-back.                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (robertbaldyga) [inline pages/open_cas_linux/guide_terminology.md]: The 'lazy write' is used in this documentation once, in description of WB, to which we redirect here...

**Comment by felipe-barajas** [comment]
Commit 59616974e84549b4b35d308322778b17268a7e52 addresses @robertbaldyga comments above

File: pages/open_cas_linux/guide_advanced_options.md [hunk 7]
```diff
@@ -0,0 +1,356 @@
...
+Kernel Module Parameters
+------------------------
+
+Following is a list of available system and kernel parameters that can be
+modified to further improve the performance of certain applications. These
+parameters should not be modified without advanced tuning expertise and a
+thorough knowledge of the workload. Please contact the Open CAS community  for further
+details or support.
+
+The parameters are: unaligned_io, seq_cut_off_mb, metadata_layout, and
```
Comment (michalwy) [inline pages/open_cas_linux/guide_advanced_options.md]: There's missing description for seq_cut_off_mb parameter.

File: pages/open_cas_linux/guide_introduction.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+| DT generic data testing program | <https://github.com/RobinTMiller/dt>                    |
+| FIO                             | <https://github.com/axboe/fio>                                    |
+| Vdbench                         | [http://vdbench.sourceforge.net](http://vdbench.sourceforge.net/) |
+
+Revision History
+----------------
+
+| **Revision Number** | **Description**                                                | **Date**       |
+|---------------------|----------------------------------------------------------------|----------------|
+| 01                  | Initial Release based on CAS v3.8 GA (GM)                      | April 2018     |
```
Comment (michalwy) [inline pages/open_cas_linux/guide_introduction.md]: Please remove reference to CAS 3.8 GA. There's no such OpenCAS version.

File: pages/open_cas_linux/guide_terminology.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+| core device        | The device to be cached.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| dirty data         | This refers to data that is modified within the cache but not modified in main memory.                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```
Comment (michalwy) [inline pages/open_cas_linux/guide_terminology.md]: I would change "main memory" to "primary storage" or similar term.

**Pull request merged**